src/components/SystemIntakeReview/index.tsx

```diff
@@ -12,11 +12,15 @@ import { yesNoMap } from 'data/common';
 import { SystemIntakeForm } from 'types/systemIntake';
 import convertBoolToYesNo from 'utils/convertBoolToYesNo';
 
-type SystemIntakeReview = {
+type SystemIntakeReviewProps = {
```

Comment:
Reusing the same name was causing a linting error. I appended the `Props` suffix to match the general convention in our codebase.